js/controller.js

```diff
@@ -1293,10 +1293,16 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       }
       // no support
       else {
-        if(this.state.isFullWindow) {
-          this.exitFullWindow();
+
+        if(OO.isIos && this.videoVr){
```

Comment:
Do we need a unit test for this condition?

Reply:
Ok. Fix